src/components/button-group/ButtonGroup.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+import './ButtonGroup.scss';
+
+export interface ButtonGroupProps {
+    /** A group of buttons */
+    children: React.ReactNode;
+    /** Class name for ButtonGroup */
+    className?: string;
+    isDisabled?: boolean;
```

Comment:
Probably add a doc for this too

Reply:
@priyajeet Done! Can you review this again?